### ISSUE TITLE
Symbol on Print - rotation

### ISSUE THREAD

**Issue by svagap** [camptocamp/ngeo#3750]
Symbols on rotated prints are not oriented correct

![onscreenwithrotationpreview](https://user-images.githubusercontent.com/29245949/38613349-f1bc1c02-3d89-11e8-93fc-3aa7c2f3217b.PNG)
Symbols on Screen without print rotation

![onscreennorotationpreview](https://user-images.githubusercontent.com/29245949/38613356-fa4c3fa0-3d89-11e8-8068-49f46ae71348.PNG)
Symbols onScreen with print rotation (they align with the lines)

![resultprintwithrotation](https://user-images.githubusercontent.com/29245949/38613461-50d10c48-3d8a-11e8-94a8-5f97e7173ebe.png)
Symbols are not aligned anymore

**Comment by svagap** [comment]
It is a bit annoying, not to get the smallest reaction of anybody only if the problem is not exactly written as expected. It would take mutch time to at leasst write a comment like "please provide more info" or "can not reproduce issue" or at least "We have read it, we do not understand" or "works as designed"

**Comment by sbrunner** [comment]
For that you should add the the 'useNativeAngle' boolean metadata in your `available_metadata` present in your project vars file:
```yaml
...
available_metadata:
  - name: printNativeAngle
    type: boolean
```

Then in the admin interface you should set this metadata to false on your first level layer group.

**Comment by rbovard** [comment]
@sbrunner Is it a workaround for https://github.com/mapserver/mapserver/issues/5271 ?

**Comment by sbrunner** [comment]
Yes but mapserver/mapserver#5271 will solve the issue relay better :-)

**Comment by sbrunner** [comment]
Then you should do a support ticket...

**Comment by svagap** [comment]
I will do a support ticket if you prove that it works in your environment

**Comment by ybolognini** [comment]
The MapServer issue (https://github.com/mapserver/mapserver/issues/5271) has been solved as part of another project. It is now being tested and will be included in release 7.2 of MapServer.
